Modules/posixmodule.c

```diff
@@ -7242,7 +7242,7 @@ os_symlink_impl(PyObject *module, path_t *src, path_t *dst,
 
     if (windows_has_symlink_unprivileged_flag) {
         Py_BEGIN_ALLOW_THREADS
-        check_flag = !result && ERROR_INVALID_PARAMETER == GetLastError()
+        check_flag = !result && ERROR_INVALID_PARAMETER == GetLastError();
```

Comment:
What is the purpose of having a separate `check_flag` and why is it computed after `Py_BEGIN_ALLOW_THREADS`? If the flag is supported, we'll have a no-op `Py_BEGIN_ALLOW_THREADS / Py_END_ALLOW_THREADS` here.